tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/mailet/OpenPaasAmqpForwardAttribute.java

```diff
@@ -0,0 +1,112 @@
+package com.linagora.tmail.mailet;
+
+import static org.apache.james.backends.rabbitmq.Constants.DURABLE;
+
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import jakarta.mail.MessagingException;
+
+import org.apache.james.backends.rabbitmq.ReactorRabbitMQChannelPool;
+import org.apache.mailet.Attribute;
+import org.apache.mailet.AttributeValue;
+import org.apache.mailet.Mail;
+import org.apache.mailet.MailetConfig;
+import org.apache.mailet.MailetException;
+import org.apache.mailet.base.GenericMailet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.fge.lambdas.Throwing;
+import com.linagora.tmail.OpenPaasModule;
+import com.rabbitmq.client.AlreadyClosedException;
+import com.rabbitmq.client.ShutdownSignalException;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.rabbitmq.ExchangeSpecification;
+import reactor.rabbitmq.OutboundMessage;
+
+public class OpenPaasAmqpForwardAttribute extends GenericMailet {
+    private static final Logger LOGGER = LoggerFactory.getLogger(OpenPaasAmqpForwardAttribute.class);
+
+    private final ReactorRabbitMQChannelPool openPaasRabbitChannelPool;
+    private OpenPaasAmqpForwardAttributeConfig config;
+
+    @Inject
+    public OpenPaasAmqpForwardAttribute(@Named(OpenPaasModule.OPENPAAS_INJECTION_KEY) ReactorRabbitMQChannelPool openPaasRabbitChannelPool) {
+        this.openPaasRabbitChannelPool = openPaasRabbitChannelPool;
+        openPaasRabbitChannelPool.start();
```

Comment:
Do we need check `isStart` first? before invoke `.start` 
I'm not sure about ReactorRabbitMQChannelPool api, 
but concern if the `.start` was invoked several time.
// ignore if it correct

Reply:
Makes sense, I updated the code to only start the channel pool (create the sender), when getSender is null (the start method was not called before).

Reply:
This is actually necessary because `.start` only creates a sender. However, we cannot verify if the sender is closed or not, and there is no API to check this, so we need to create a new sender each time.